internal/gatewayapi/testdata/httproute-with-single-rule-with-multiple-rules.out.yaml

```diff
@@ -92,11 +92,16 @@ httpRoutes:
     parents:
     - conditions:
       - lastTransitionTime: null
-        message: 'Regex "*regex*" is invalid: error parsing regexp: missing argument
-          to repetition operator: `*`.'
+        message: Route is accepted
+        reason: Accepted
+        status: "True"
+        type: Accepted
+      - lastTransitionTime: null
+        message: 'Failed to process route rule 2: regex "*regex*" is invalid: error
+          parsing regexp: missing argument to repetition operator: `*`.'
         reason: UnsupportedValue
         status: "False"
-        type: Accepted
```

Comment:
this doesnt look right
https://github.com/kubernetes-sigs/gateway-api/blob/main/geps/gep-1364/index.md?plain=1